www/global-configs.js

```diff
@@ -3,6 +3,8 @@ var globalConfigs = {
   serializer: 'urlencoded',
   followRedirect: true,
   timeout: 60.0,
+  connectTimeout: 30.0,
+  readTimeout: 30.0
```

Comment:
I think it's better to use 60 seconds for both as default values to be consistent with older versions.

Reply:
Okie agreed